pyproject.toml

```diff
@@ -37,13 +37,26 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Application Frameworks",
 ]
 dependencies = [
+    # Django version support is added one minor release at a time
     "Django>=2.1,<4.2",
-    "bleach[css]>=5",
+    # bleach has been pretty stable, hence the loose pin
+    "bleach[css]>=5,<6",
+    # Pillow is used very little and has never broken
     "Pillow",
```

Comment:
Shouldn't we also pin the Pillow version?

Reply:
Could be good, I don't have strong opinions on this one... I don't even know what the lower bound should be.

Reply:
I'll leave this one as it is, bcz most projects already have/use a pinned Pillow version and Django-Wiki will use that version (if its in the right order).